dwellingplace/models/_utils.py

```diff
@@ -92,17 +95,29 @@ def save_xlsx(data: list, sheets: List[str], path: str):
         for datum in data:
 
             if skip_row(datum, header):
-                log.debug("Skipped empty row: %s", datum)
                 continue
 
-            log.debug("Add row: %s", datum)
+            update_cache(datum, prefill_cache)
+
             for col, key in enumerate(header):
                 value, options = get_value(datum, key)
                 fmt = workbook.add_format(options) if options else None
                 worksheet.write(row, col, value, fmt)
 
             row += 1
 
+        # Prefill rows
+        if prefill and prefill_cache:
+            most_recent_date = max(prefill_cache.keys())
+            next_date = increment_month(most_recent_date)
+            log.info("Prefilling data for %s", next_date)
+            for key in prefill_cache[most_recent_date]:
+                worksheet.write(row, 0, key)
+                # TODO: don't specify formatting here, use `get_value`
```

Comment:
Yeah, it needs to be cleaned up if we have time, but I'm going to merge this now.